packages/node_modules/@webex/widget-meet/src/container.js

```diff
@@ -50,11 +50,20 @@ export const ownPropTypes = {
 export class MeetWidget extends Component {
   constructor() {
     super();
-    this.state = {};
+    this.state = {
+      escapedAuthentication : false
+    };
   }
 
-  shouldComponentUpdate(nextProps) {
-    return nextProps !== this.props;
+  
+  setEscapedAuthentication = (updatedValue) => {
```

Comment:
Why does it have to be done this way? Why not directly call `this.setState`?

Reply:
Is this because it is being passed as a props to another component? Makes sense to write a callback like this to me. @Shreyas281299 can confirm.

Reply:
@rarajes2  you're right. SInce this is being asked as a props I wrote a callback. @sreenara